service-front/module/Application/test/Controller/CPFlowControllerTest.php

```diff
@@ -37,19 +38,133 @@ public function setUp(): void
         $serviceManager->setService(FormProcessorHelper::class, $this->formProcessorService);
     }
 
-    public function testCPIdCheckReturnsPageWithData(): void
+    public function returnOpgResponseData(): array
     {
-        $mockResponseDataIdDetails = [
-            "FirstName" => "Mary Anne",
-            "LastName" => "Chapman",
-            "DOB" => "01 May 1943",
-            "Address" => "Address line 1, line 2, Country, BN1 4OD",
-            "Role" => "cp",
-            "LPA" => [
-                "PA M-1234-ABCB-XXXX",
-                "PW M-1234-ABCD-AAAA"
+        return [
+            "id" => "2d86bb9d-d9ce-47a6-8447-4c160acaee6e",
+            "personType" => "certificateProvider",
+            "firstName" => "Mary Anne",
+            "lastName" => "Chapman",
+            "dob" => "01 May 1943",
+            "address" => [
+                "1 Court Street",
+                "London",
+                "UK",
+                "SW1B 1BB",
+            ],
+            "lpas" => [
+                "M-XYXY-YAGA-35G3",
+            ],
+            "documentComplete" => false,
+            "alternateAddress" => [
+            ],
+            "selectedPostOfficeDeadline" => null,
+            "selectedPostOffice" => null,
+            "searchPostcode" => null,
+            "idMethod" => "nin"
+        ];
+    }
+
+    public function returnSiriusLpaResponse(): array
+    {
+        return [
+            "opg.poas.lpastore" => [
+                "attorneys" => [
```

Comment:
It seems unnecessary to include all this data in the mock response when we don't use 90% of it. Could we reduce the response to just the bits that are needed?